src/scp.c

```diff
@@ -273,6 +273,7 @@ scp_recv(LIBSSH2_SESSION * session, const char *path, libssh2_struct_stat * sb)
     size_t cmd_len;
     int rc;
     int tmp_err_code;
+    size_t path_len;
```

Comment:
Can we move this to the `if()` block that uses the variable?

Reply:
Done, pushed a second ago.